### DISCUSSION TITLE
Can I apply cursor trail from Argon to another skin?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#20954] [Q&A] [answered]
I like this cursortrail, and I want to play with similar trail on another skin. Is it possible?

**Comment by peppy** [comment, marked as answer]
Not currently. Customising cursor trails will come later.

**Reply by okolenmi** [reply]
Thank you for answer. It sounds great. This type of cursortrail is much easier to read on fast maps. It would also be great to be able to change its color.
